core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -5002,7 +5002,7 @@ protected function listView()
 <table class="tl_listing' . (($GLOBALS['TL_DCA'][$this->strTable]['list']['label']['showColumns'] ?? null) ? ' showColumns' : '') . ($this->strPickerFieldType ? ' picker unselectable' : '') . '">';
 
 			// Automatically add the "order by" field as last column if we do not have group headers
-			if ($GLOBALS['TL_DCA'][$this->strTable]['list']['label']['showColumns'] ?? null)
+			if (($GLOBALS['TL_DCA'][$this->strTable]['list']['label']['showColumns'] ?? null) && ($GLOBALS['TL_DCA'][$this->strTable]['list']['label']['showFirstOrderBy'] ?? true))
```

Comment:
```suggestion
			if (($GLOBALS['TL_DCA'][$this->strTable]['list']['label']['showColumns'] ?? null) && !($GLOBALS['TL_DCA'][$this->strTable]['list']['label']['hideFirstOrderBy'] ?? false))
```

I think an explicit `true` is better than an implicit `false`.

Reply:
I'd prefer a setting that enables something when `true` and disables when `false`. I find reverse settings confusing and counter-intuitive, and Contao already has enough of them. But I get your point. Do you insist on `hideFirstOrderBy`, or is it up for discussion?

Reply:
> I'd prefer a setting that enables something when `true` and disables when `false`.

Then the code should be `&& false !== ($GLOBALS['TL_DCA'][$this->strTable]['list']['label']['showFirstOrderBy'] ?? true)`, which would make it explicit again.

> Do you insist on `hideFirstOrderBy`, or is it up for discussion?

Everything is up for discussion. 😄